frontend/test/playwright/utils/expect-snapshot.ts

```diff
@@ -0,0 +1,110 @@
+import { expect } from "@playwright/test"
+
+import { type LanguageDirection, t } from "~~/test/playwright/utils/i18n"
+
+import type { Breakpoint } from "~/constants/screens"
+
+import type {
+  Expect,
+  Locator,
+  LocatorScreenshotOptions,
+  Page,
+  PageScreenshotOptions,
+} from "@playwright/test"
+
+export type ExpectSnapshotOptions = {
+  screenshotOptions?: LocatorScreenshotOptions | PageScreenshotOptions
+  snapshotOptions?: Parameters<ReturnType<Expect>["toMatchSnapshot"]>[0]
+  dir?: LanguageDirection
+  useColorMode?: boolean
+}
+export type ExpectSnapshot = <T extends Locator | Page>(
```

Comment:
nit: Whitespace to help break up the file's components

```suggestion
}

export type ExpectSnapshot = <T extends Locator | Page>(
```